opta/core/azure.py

```diff
@@ -170,3 +172,38 @@ def bucket_exists(self, bucket_name: str) -> bool:
             return True
         except ResourceNotFoundError:
             return False
+
+    @classmethod
+    def get_detailed_config_map(cls) -> Dict[str, Dict[str, str]]:
```

Comment:
every line of code that we add cost us in maintenance, so since the config command doesn't support azure, we should not keep the implementation.

Reply:
Removed Azure support for now.